manon/js/language-selector.js

```diff
@@ -14,43 +14,60 @@ function initLanguageSelector() {
   });
 }
 
-function onClick(e) {
-  var languageSelectorElement = e.target.closest(".language-selector-options");
+/**
+ * @param {Event} event
+ */
+function onClick(event) {
+  if (!(event.target instanceof HTMLElement)) return;
```

Comment:
Should we add a check on the event type? The `onKeyPress` also checks the event type.

```suggestion
  if (!(event instanceof PointerEvent)) return;
  if (!(event.target instanceof HTMLElement)) return;
```

Got the event type from: https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event